spec/mongo/cluster/app_metadata_spec.rb

```diff
@@ -112,7 +112,17 @@
         end
 
         it 'truncates the document to be just an ismaster command and the compressors', unless: compression_enabled? do
-          expect(app_metadata.ismaster_bytes.length).to eq(Mongo::Server::Monitor::Connection::ISMASTER_BYTES.length + 26)
+          sasl_supported_mechs_size = 0
```

Comment:
Given that we sometimes query the server for the supported authentication mechanisms, we need to make sure that when we assert the length of the ismaster document that we include that info.

Reply:
This one